static/app/utils/gettingStartedDocs/python.spec.tsx

```diff
@@ -0,0 +1,63 @@
+// Mock the modules that cause circular dependencies
+jest.mock('sentry/utils/gettingStartedDocs/python', () => {
+  const original = jest.requireActual('sentry/utils/gettingStartedDocs/python');
+  return {
+    ...original,
+    // Mock any functions causing circular dependencies
+    getPythonProfilingOnboarding: jest.fn(),
+  };
+});
+
+import {getPythonInstallSnippet} from 'sentry/utils/gettingStartedDocs/python';
+
+describe('getPythonInstallSnippet', () => {
+  it('generates pip install command with default parameters', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk',
+    });
+
+    expect(result.pip).toBe(`pip install "sentry-sdk"`);
+  });
+
+  it('generates pip install command with minimum version and extras', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk[with-extras]',
+      minimumVersion: '1.2.3',
+    });
+    expect(result.pip).toBe(`pip install --upgrade "sentry-sdk[with-extras]>=1.2.3"`);
+  });
+
+  it('generates uv install command with default parameters', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk',
+    });
+
+    expect(result.uv).toBe(`uv add "sentry-sdk"`);
+  });
+
+  it('generates uv install command with minimum version and extras', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk[with-extras]',
+      minimumVersion: '2.3.4',
+    });
+
+    expect(result.uv).toBe(`uv add --upgrade "sentry-sdk[with-extras]>=2.3.4"`);
+  });
+
+  it('generates poetry install command with default parameters', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk',
+    });
```

Comment:
nit: 

```suggestion
    const result = getPythonInstallSnippet();
```
Isn't "sentry-sdk" the default packageName?

Reply:
Not in the InstallSnippet; only in the InstallConfig, which is why it needs to be passed in here in the test